packages/typespec-ts/test/integration/azureArmResources.spec.ts

```diff
@@ -146,7 +146,7 @@ describe("Azure Arm Resources Rest Client", () => {
     assert.strictEqual(result.body.id, validSingletonResource.id);
     assert.strictEqual(result.body.name, validSingletonResource.name);
     assert.strictEqual(result.body.type, validSingletonResource.type);
-    assert.strictEqual(result.body.location, "eastus2");
+    assert.strictEqual(result.body.location, "eastus");
```

Comment:
refer to https://github.com/Azure/cadl-ranch/pull/759/files#diff-c2ca70aeb42e6c62b6c6312570b2635edd217555e21b5597346dc30e88839e58